### PR TITLE
Users/singankit/evaluation async samples

### PR DESCRIPTION
**Pull request by singankit**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+DESCRIPTION:
+    This sample demonstrates how to use evaluation operations from
+    the Azure Evaluation service using a asynchronous client.
+
+USAGE:
+    python sample_evaluation_async.py
+
+    Before running the sample:
+
+    pip install azure.ai.projects azure-identity
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py:18]: don't you also need aiohttp?

File: sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import Evaluation, Dataset, EvaluatorConfiguration, ConnectionType
+from azure.ai.evaluation import F1ScoreEvaluator, RelevanceEvaluator, ViolenceEvaluator
+
+
+async def main():
+    # Create an Azure AI Client from a connection string, copied from your AI Studio project.
+    # At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+    # Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py:37]: I removed the above comments in my samples. And I think Agents samples as well. There will be is plenty of info on this in the package README.md.

File: sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py [hunk 3]
```diff
@@ -54,26 +53,28 @@
...
     evaluators={
         "f1_score": EvaluatorConfiguration(
-            id=F1ScoreEvaluator.evaluator_id,
+            # id=F1ScoreEvaluator.id,
+            id="azureml://registries/azureml-staging/models/F1Score-Evaluator/versions/3",
         ),
         "relevance": EvaluatorConfiguration(
-            id=RelevanceEvaluator.evaluator_id,
+            # id=RelevanceEvaluator.id,
+            id="azureml://registries/azureml-staging/models/Relevance-Evaluator/versions/3",
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py:61]: Is the line ` # id=RelevanceEvaluator.id,` needed?

File: sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py [hunk 4]
```diff
@@ -54,26 +53,28 @@
...
             init_params={
-                "model_config": default_connection.to_evaluator_model_config(deployment_name="GPT-4-Prod", api_version="2024-08-01-preview")
+                "model_config": default_connection.to_evaluator_model_config(deployment_name=deployment_name,
+                                                                             api_version=api_version)
             },
         ),
-        "hate_unfairness": EvaluatorConfiguration(
-            id=HateUnfairnessEvaluator.evaluator_id,
+        "violence": EvaluatorConfiguration(
+            # id=ViolenceEvaluator.id,
```
Comment (dargilco) [inline sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py:68]: Same

**Review by dargilco — APPROVED** [review]

File: sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py (hunk 1 above)
Comment (singankit) [reply sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py:18]: Yes that is correct it is needed

File: sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py (hunk 3 above)
Comment (singankit) [reply sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py:61]: Yes that will be the final sample. However I am waiting for evaluators to get published and azure-ai-evaluation package to reflect it.

File: sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py (hunk 1 above)
Comment (singankit) [reply sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py]: Updated thanks for catching that

File: sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py (hunk 2 above)
Comment (singankit) [reply sdk/ai/azure-ai-projects/samples/evaluations/async_samples/sample_evaluations_async.py]: Removed it.

**Pull request merged**